src/Helpers/GetUserHelper.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\BLT\Helpers;
+
+class GetUserHelper
+{
+    public static function getUser(string $field, string $value): object
```

Comment:
```suggestion
    public static function getUser(string $field, string $value): ?object
```

I think this will be better, just in case if user is not found